app/Repositories/Eloquents/UserRepository.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Repositories\Eloquents;
+
+use Auth;
+use App\Models\User;
+use App\Repositories\Eloquents\BaseRepository;
+use App\Repositories\Interfaces\UserRepositoryInterface;
+
+class UserRepository extends BaseRepository implements UserRepositoryInterface
+{
+    public function getModel()
+    {
+        return User::class;
+    }
+
+    public function getAllAndSearch($perPage, $searchParams)
+    {
+        $query = $this->model->orderBy('created_at', 'desc');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/UserRepository.php</br>**Line**: 21</br>Blank line found at start of control structure